### PR TITLE
Use Symfony stable releases

### PR DESCRIPTION
**Pull request by theofidry**
Use Symfony 2.8 et 3.0 stable releases instead of the dev one.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -31,8 +31,8 @@ matrix:
     - php: 7.0
     - php: nightly
     - php: hhvm
-    - env: SYMFONY_VERSION=2.8.*@dev
-    - env: SYMFONY_VERSION="3.0.*@dev"
+    - env: SYMFONY_VERSION=2.8.*
```
Comment (Soullivaneuh) [inline .travis.yml]: This should be simply removed from `allow_failures` and `@dev` should be removed here: https://github.com/hautelook/AliceBundle/pull/169/files#diff-354f30a63fb0907d4ad57269548329e3R27

**Comment by Soullivaneuh** [comment]
BTW, we can check if php7 and hhvm pass. If yes, they can be removed from `allowed_failure`.

We can take also profit to add SF 3.1 on `allowed_failures` section.

**Comment by theofidry** [comment]
Removed the dev ones.

>This should be simply removed from allow_failures

Once we ensured the compatibility that is.

>BTW, we can check if php7 and hhvm pass

Yeah that would be great.

>We can take also profit to add SF 3.1 on allowed_failures section

Agreed

**Comment by Soullivaneuh** [comment]
> Once we ensured the compatibility that is.

Should be OK after https://github.com/hautelook/AliceBundle/pull/173.

Note: My PRs is agains 1.x branch. Should be the same for this PR?

**Comment by theofidry** [comment]
Ah right, forgot we have two branches now. Yeah we should work on the 1.x and then port it to master.

**Comment by Soullivaneuh** [comment]
1.x is green now, I let you do the rebase and open another PR. ;-)

**Comment by theofidry** [comment]
Closed in favour of #188

**Pull request closed without merging**